pkg/volume/fc/fc.go

```diff
@@ -82,11 +82,7 @@ func (plugin *fcPlugin) GetVolumeName(spec *volume.Spec) (string, error) {
 }
 
 func (plugin *fcPlugin) CanSupport(spec *volume.Spec) bool {
-	if (spec.Volume != nil && spec.Volume.FC == nil) || (spec.PersistentVolume != nil && spec.PersistentVolume.Spec.FC == nil) {
-		return false
-	}
-
-	return true
+	return (spec.Volume != nil && spec.Volume.FC != nil) || (spec.PersistentVolume != nil && spec.PersistentVolume.Spec.FC != nil)
```

Comment:
why the format change? How did you even end up herE?

Reply:
seems catch other merged commit log[https://github.com/kubernetes/kubernetes/pull/59548](https://github.com/kubernetes/kubernetes/pull/59548)